controllers/grants.go

```diff
@@ -36,6 +36,7 @@ func GrantsRouter() chi.Router {
 		r.Post("/", middleware.InstrumentHandlerFunc("RedeemGrants", RedeemGrants))
 	}
 	r.Put("/{grantId}", middleware.InstrumentHandlerFunc("ClaimGrant", ClaimGrant))
+	r.Get("/check", middleware.InstrumentHandlerFunc("CheckGrant", CheckGrant))
```

Comment:
I think you can remove this (and the below CheckGrant function) given you're adding the info about already redeemed grants to the response of the existing redeem endpoint